lib/rouge/demos/ttcn3

```diff
@@ -0,0 +1,6 @@
+module DNSTester {
```

Comment:
The demo is intended to be only a couple of lines that briefly test the lexer. I've used only a couple of lines from your sample file.